agent/consul/config_replication.go

```diff
@@ -214,9 +217,10 @@ func (s *Server) replicateConfig(ctx context.Context, lastRemoteIndex uint64, lo
 			return 0, true, nil
 		}
 		if err != nil {
-			return 0, false, fmt.Errorf("failed to delete local config entries: %v", err)
+			merr = multierror.Append(merr, err)
```

Comment:
do we want to preserve the error message here?

Reply:
the inner errors will be of the form `Failed to apply config entry <delete|upsert>: <error with details>` so since the delete/update portion was already present I didn't see the value.